sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -138,22 +139,35 @@ export class InstitutionController extends BaseController {
       );
     }
 
-    // Get user details
-    const accountDetails = await this.accountService.getAccountDetails(
-      body.userId,
-    );
-    if (!accountDetails) {
-      throw new UnprocessableEntityException(
-        `Unable to account detail of user ${body.userId}`,
-      );
+    let userEntity: User;
+
+    if (body.userGuid) {
+      userEntity = await this.userService.getUser(`${body.userGuid}@bceid`);
     }
 
-    // Create User
-    const userEntity = this.userService.create();
-    userEntity.email = accountDetails.user.email;
-    userEntity.firstName = accountDetails.user.firstname;
-    userEntity.lastName = accountDetails.user.surname;
-    userEntity.userName = `${accountDetails.user.guid}@bceid`;
+    if (!userEntity) {
```

Comment:
Can you please elaborate in which scenario the userEntity will be null at this stage?